content/soda-gettingstarted/quickstart.md

```diff
@@ -11,9 +11,10 @@ This is a quickstart guide for users and new contributors to try and get familia
 To have more control of the installaton cofigurations, or try out various configurations refer a detailed installation guide [here](/soda-gettingstarted/installation)
 
 ## Quick Installation and experience of SODA releases
-### Install SODA v1.0.0 Faroe Release Jun 2020
 
-Quick installation from SODA Faroe Release:
+### Install SODA v1.1.0 Greenland Release September 2020
+Quick installation from SODA Greenland Release:
+
 
```

Comment:
We add a section to mention the releases supported with bulletins. Reverse chronological order
Greenland
Faroe
Elba
Then continue the quick start info\

Please remove the elba from the past release section and update properly

Reply:
Removed Elba from Past release section.